rules/stylistic.js

```diff
@@ -0,0 +1,30 @@
+export default {
+  rules: {
+    'multiline-comment-style': 'error',
```

Comment:
same for this one, it was previously included by eslint, but now is part of the stylistic plugin